scripts/AppScriptTemplate

```diff
@@ -62,8 +62,8 @@ CONFIGNAME=""
 while [ $# -gt 0 ] ; do
     case $1 in
         -d|--debug) DEBUG="yes" ;;
-        --profile=*) JMRI_OPTIONS="${JMRI_OPTIONS} -Dorg.jmri.profile=\"${1#*=}\"" ;;
-        -p) JMRI_OPTIONS="${JMRI_OPTIONS} -Dorg.jmri.profile=\"$2\"" ; shift ;;
+        --profile=*) JMRI_OPTIONS="${JMRI_OPTIONS} -Dorg.jmri.profile=${1#*=}" ;;
```

Comment:
Profile names can contain spaces, but IDs cannot. An ID is a normalized profile name with an eight digit random hexadecimal number (this allows two profiles to have the same name, with unique IDs, without relying on the user to ensure uniqueness).

Because of this, the argument must use the ID instead of the name.